packages/py3_twine.rb

```diff
@@ -1,27 +1,27 @@
-# Adapted from Arch Linux python-twine PKGBUILD at:
-# https://aur.archlinux.org/cgit/aur.git/plain/PKGBUILD?h=python-twine
-
+# Twine 5.1.1 currently breaks on some uploads.
+# Install twine from https://github.com/pypa/twine/pull/1123 instead.
 require 'buildsystems/pip'
 
 class Py3_twine < Pip
   description 'A utility for interacting with PyPI'
   homepage 'https://pypi.python.org/pypi/twine'
-  version '5.1.1-py3.12'
+  version '5.1.2.dev26+gc577765-py3.12'
```

Comment:
Why all this instead of just a commit hash and a url patch?

Reply:
Because it's much easier to have pip install directly from git than actually download the source and patch ourselves.

This way we just follow the relevant patch branch...